README.md

```diff
@@ -89,14 +89,20 @@ You can also use the container described below.
 Running a local instance of the transitous setup can be useful for debugging.
 The easiest way is to use the same container image that we use for fetching and importing the data on the CI.
 
-First, build the container:
+First, ensure that you have the Git submodules:
+
+```bash
+git submodule init && git submodule update
+```
+
+Proceed by building the container:
 ```bash
 podman build ci/container/ -t transitous -f ci/container/Containerfile
 ```
 
 Enter the container:
 ```bash
-podman run -it -p 8080:8080 -v $PWD:/transitous -w /transitous transitous
+podman run -it --privileged -p 8080:8080 -v $PWD:/transitous -w /transitous transitous
```

Comment:
To ensure that within the container the user has enough privileges (root). Since the `ci/fetch-feeds.py` is owned, if the container is run as a normal user you need to be root within the container to run it

Reply:
Actually, a different way might be better

Reply:
What distro are you using? My assumption is that in your distro the `--userns=keep-id` gets added with an alias or config at every run. If you try to do an `ls -alhZ` in the container, who owns the files? if no `--userns=keep-id` is used, they should be owned by root

Reply:
There are two different situations here:
1. Being on Debian, you have no SELinux running, so since `fetch.py` has `o+rx` it can be executed. If you have SELinux running that's not enough (hence the `:Z`)
2. The `transitous` user probably has enough privileges to write `/transitous/out` and `/transitous/downloads`, which is probably due to how those were setup on your system. Usually folders get created as 755 in $HOME which would not be enough. The `--userns=keep-id` allows to avoid adding `o+w` for those folders

Reply:
It should not be a problem, since Podman should drop all parameters/options that are not applicable to the current system (eg: the `Z` on a system without SELinux should have no effect)